mmcv/ops/nms.py

```diff
@@ -37,14 +50,26 @@ def symbolic(g, bboxes, scores, iou_threshold, offset):
             from torch.onnx.symbolic_opset9 import select, squeeze, unsqueeze
             boxes = unsqueeze(g, bboxes, 0)
             scores = unsqueeze(g, unsqueeze(g, scores, 0), 0)
-            max_output_per_class = g.op(
-                'Constant',
-                value_t=torch.tensor([sys.maxsize], dtype=torch.long))
+
+            from torch.onnx.symbolic_helper import _is_value, _size_helper
+            if not _is_value(max_num):
```

Comment:
Could you update the test unit of `nms` and `batched _nms` in `test_onnx.py` and `test_tensorrt.py` if possible.